client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -132,7 +117,7 @@ const CalendarPage = () => {
       <p className={styles.calendarHeader}>
         {dailyTaskDateFormatter(currDate)}
       </p>
-      <ul className={styles.taskList}>{cards}</ul>
+      <TaskListView tasks={children} taskEdittable={true} />
```

Comment:
I would use a more descriptive name than `children`. Perhaps `taskCards` if that isn't already used

Reply:
Renamed to `sampleTasks`